Jenkins-Repro-Steps.md

```diff
@@ -40,6 +42,17 @@ Repro with these commands:
 ./test/runtests.sh -d
 ```
 
+### OSX Repro Steps
+
+Repro with these commands:
+
+```
+./build.sh --debug --static --icu=/usr/local/opt/icu4c/include -j=2
```

Comment:
Oh, should we add the additional set of non-static build configs for OSX on `master`?